javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,15 @@
+var fullName = 'Youssef Najjarine';
+var isCool = false;
+var totalPets = 1;
+var unknown = null;
+var never =
```

Comment:
This is a syntax error, please run your code before submitting.

Reply:
![image](https://user-images.githubusercontent.com/71291742/102308050-a41d1100-3f1a-11eb-8903-4c357ed8abfd.png)

It ran fine for me, and in the instructions it said to not assign it anything
![image](https://user-images.githubusercontent.com/71291742/102308181-f9f1b900-3f1a-11eb-9acf-e94e1e3e01c5.png)